lib/vscode/src/vs/code/browser/workbench/workbench.ts

```diff
@@ -495,14 +503,8 @@ class WindowIndicator implements IWindowIndicator {
 	// Finally create workbench
 	create(document.body, {
 		...config,
-<<<<<<< HEAD
-=======
-		developmentOptions: {
```

Comment:
We might want to set `developmentOptions: { logLevel }` in our `config` here:

https://github.com/cdr/code-server/blob/vscode-1.56/lib/vscode/src/vs/server/node/server.ts#L97

Reply:
Actually might not be a bad idea to link that file in a comment here so it's obvious where `config`  comes from.